react/global.css

```diff
@@ -1,65 +1,65 @@
-/* General */
-.vtex-carousel{
+/* General */	
+.vtex-carousel{	
```

Comment:
if is on CSS modules then remove from global

Reply:
Done.